src/log.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef CNT4007_LOG_HPP
+#define CNT4007_LOG_HPP
+
+#include <fstream>

```

Comment:
See system.hpp, all the standard library header files are there. Then, just include system.hpp and you'll automatically have access to all the libraries. This is just so we can keep includes organized:

Instead of 
```c++
#include <fstream>
```

Do:
```c++
#include "system.hpp"
```